### PR TITLE
Add `EXTERNAL_IMPORT_ENABLED` environment variables

### PR DESCRIPTION
**Pull request by 196Ikuchil**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

Add `EXTERNAL_IMPORT_ENABLED` environment variables it indicates whether the scheduler should enable the function to import from an existing cluster.
Eventually, we would like to implement https://github.com/kubernetes-sigs/kube-scheduler-simulator/issues/41 .

#### Which issue(s) this PR fixes:

Relates #41

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -16,12 +16,14 @@ import (
 var ErrEmptyEnv = errors.New("env is needed, but empty")
 
 // Config is configuration for simulator.
+// ExternalImportEnabled indicates whether the scheduler should enable the function to import from an existing cluster.
```
Comment (sanposhiho) [inline config/config.go:19]: Let's move on to `ExternalImportEnabled` field. (between L24 and L25)

**Review by sanposhiho — COMMENTED** [review]
Thanks! One nits:

**Comment by 196Ikuchil** [comment]
Thank you for the review! I fixed it.

File: config/config.go [hunk 2]
```diff
@@ -17,11 +17,13 @@ var ErrEmptyEnv = errors.New("env is needed, but empty")
...
-	Port                int
-	KubeAPIServerURL    string
-	EtcdURL             string
-	FrontendURL         string
-	InitialSchedulerCfg *v1beta2config.KubeSchedulerConfiguration
+	Port             int
+	KubeAPIServerURL string
+	EtcdURL          string
+	FrontendURL      string
+	// ExternalImportEnabled indicates whether the scheduler should enable the function to import from an existing cluster.
```
Comment (sanposhiho) [inline config/config.go:24]:
```suggestion
	// ExternalImportEnabled indicates whether the simulator will import resources from an existing cluster or not.
```

File: config/config.go [hunk 3]
```diff
@@ -131,6 +136,14 @@ func getSchedulerCfg() (*v1beta2config.KubeSchedulerConfiguration, error) {
 	return sc, nil
 }
 
+// getExternalImportEnabled reads EXTERNAL_IMPORT_ENABLED.
+// This function will be return true only if that is "1".
+// This is means External Import is disabled, by default.
```
Comment (sanposhiho) [inline config/config.go:141]:
```suggestion
// getExternalImportEnabled reads EXTERNAL_IMPORT_ENABLED and convert it to bool.
// This function will return `true` if `EXTERNAL_IMPORT_ENABLED` is "1".
```

**Review by sanposhiho — COMMENTED** [review]
Only two nits

**Comment by 196Ikuchil** [comment]
@sanposhiho Thanks! Fixed it.

**Review by sanposhiho — APPROVED** [review]
Thanks!
/lgtm
/approve

**Pull request merged**